### PR TITLE
Add persian to documenting.rst

### PR DESCRIPTION
**Pull request by mmdbalkhi**
Add the Persian language. Of course, I'm not sure if other people are working on the Persian language

### CODE REVIEW

File: documenting.rst [hunk 1]
```diff
@@ -1668,6 +1670,7 @@ in production; others are works in progress.
 .. _doc_zh_cn: https://docs.python.org/zh-cn/
 .. _doc_zh_tw: https://docs.python.org/zh-tw/
 .. _github_ar: https://github.com/Abdur-rahmaanJ/python-docs-ar
+.. _github_fa: https://github.com/mmdbalkhi/python-docs-fa
 .. _github_bn_in: https://github.com/python/python-docs-bn-in
 .. _github_es: https://github.com/python/python-docs-es
 .. _github_fr: https://github.com/python/python-docs-fr
```
[the remark this answers is not in the document]
Comment (mmdbalkhi) [reply documenting.rst]: thank you @hugovk

File: documenting.rst [hunk 2]
```diff
@@ -1602,6 +1602,8 @@ in production; others are works in progress.
 | spoken in       |                               |                            |
 | India (bn_IN)   |                               |                            |
 +-----------------+-------------------------------+----------------------------+
+| Persian (fa)     | `Komeil Parseh (mmdbalkhi)  | `GitHub <github_fa_>`_     |
++-----------------+-------------------------------+----------------------------+
```
[the remark this answers is not in the document]
Comment (hugovk) [reply documenting.rst]: The others are sorted by language name, not ISO. See for example the end of the table:

```
| Russian (ru)    |                               | `mail <mail_ru_>`_         |
+-----------------+-------------------------------+----------------------------+
| Simplified      | `Shengjing Zhu <bpo_zhsj_>`_  | `Transifex <tx_zh_cn_>`_   |
| Chinese         |                               | `GitHub <github_zh_cn_>`_  |
| (zh-cn)         |                               | `Doc <doc_zh_cn_>`_        |
+-----------------+-------------------------------+----------------------------+
| Spanish (es)    | Raúl Cumplido                 | `GitHub <github_es_>`_     |
+-----------------+-------------------------------+----------------------------+
| Traditional     | `王威翔 Matt Wang             | `GitHub <github_zh_tw_>`_  |
| Chinese         | <bpo_mattwang44_>`_,          | `Transifex <tx_zh_tw_>`_   |
| (zh-tw)         | Josix Wang                    | `Doc <doc_zh_tw_>`_        |
+-----------------+-------------------------------+----------------------------+
| Turkish (tr)    | `Ege Akman (egeakman)         | `GitHub <github_tr_>`_     |
|                 | <email_egeakman_>`_           |                            |
+-----------------+-------------------------------+----------------------------+
```

[the remark this answers is not in the document]
Comment (mmdbalkhi) [reply documenting.rst]: > The others are sorted by language name, not ISO. See for example the end of the table

ok, Thanks for the help

**Review by hugovk — APPROVED** [review]
Thanks for the PR and translation!

**Pull request merged**